src/frontend/org/voltdb/sysprocs/org/voltdb/calciteutils/CreateTableUtils.java

```
@@ -87,6 +87,17 @@
  * Util class to help with "CREATE TABLE" DDL statements through Calcite.
  */
 public class CreateTableUtils {
+
+    /**
+     * Mimick user-observable error message from VoltCompiler.VoltCompilerException, by
+     * clearing stack traces.
+     */
+    private static final class DDLException extends PlanningErrorException {
+        public DDLException(String msg) {
+            super(msg);
```

You could invoke super(String, Throwable, boolean, boolean) which allows you to disable the stack trace and suppressing exceptions. `super(msg, null, true, false);`